libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-overview.component.ts

```diff
@@ -28,6 +28,7 @@ import { LetModule } from '@rx-angular/template';
 import { DhMeteringPointDataAccessApiStore } from '@energinet-datahub/dh/metering-point/data-access-api';
 import { WattSpinnerModule } from '@energinet-datahub/watt';
 
+import { UiSecondaryMasterDataComponentScam } from './ui-secondary-master-data/ui-secondary-master-data.component';
```

Comment:
suggestion: Quick comment before continuing with more thorough review - in order to be consistent with the file naming, let's remove the `ui-` prefix from both folder, components and class names. The `ui-` prefix is used only when creating libraries that contain components used in the UI. The components themselves don't have to use that prefix.

Reply:
I have changed the prefix to dh and removed the prefix from the folder